scripts/step-3.sh

```diff
@@ -47,11 +47,11 @@ checkout_branch
 # Skip if only timestamping
 if [ -z "$DISABLE_SNAPSHOT" ]; then
     ./tuf snapshot -repository "$REPO"
-    ./tuf sign -repository "$REPO" -roles snapshot -key "${SNAPSHOT_KEY}"
+    ./tuf sign -repository "$REPO" -roles snapshot -key "${SNAPSHOT_KEY}" -add-old-type true
```

Comment:
This (and the line below) updates timestamp and snapshot to generate key ids using both new and old format.